Feature 1

tkz96 opened the pull request:
Completed the features as requested on the activity page.
Added the index.html file, styles.css file, linter, and other files as requested.

Ignored test.md

prolajumokeoni commented on styles.css:
```diff
@@ -0,0 +1,3 @@
+.h1 {
```
- [ ] Kindly remove "." before the h1 tag to enable you style directly using the h1 tag or add a class to your h1 tag in index.html and style in CSS file.

prolajumokeoni commented on README.md:
```diff
@@ -0,0 +1,50 @@
+
+# Hello Microverse
+
+> Description the project.
```
- [ ] Kindly add a description to your readme, this helps document your project professionally 👍🏿

prolajumokeoni commented on README.md:
```diff
@@ -0,0 +1,50 @@
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
- [ ] Kindly link MIT.md page, this link is broken 🤗

prolajumokeoni reviewed (changes requested):
## Hi @tkz96 ,

Good job creating your first project 👏🏿 👏🏿

There are some issues that you still need to work on to get this project approved but you are almost there 💯 

## Highlights
- Good PR ✔️ 
- Your commit messages are good ✔️ 
- Gitflow was correctly used ✔️ 

## Required Changes ♻️
- Kindly check all inline comments and implement all required changes 😃 
- Setup [Linters](https://github.com/microverseinc/linters-config/tree/master/html-css) for this project, you only added the workflows folder
- Kindly add a [test.md ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) file ignored by git as stated in the requirements.

**_Check comments under the review._**

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.
- Please check the inline comments 🤗 

**Cheers and Happy coding!👏👏👏**

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear._

Please, remember to tag me in your question here so I can receive the notification if you have any questions @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

tkz96 commented:
@prolajumokeoni
I have made all the required changes as asked.

**About test.md:**
I have already added the file to the project. However, the file is not being tracked as I have added it in the `.gitignore` file as required. 

I will make further changes if required.

Thank you. 😃

prolajumokeoni commented:
> @prolajumokeoni I have made all the required changes as asked.
> 
> **About test.md:** I have already added the file to the project. However, the file is not being tracked as I have added it in the `.gitignore` file as required.
> 
> I will make further changes if required.
> 
> Thank you. smiley

Hi @tkz96 ,

- [ ] Don't forget to add node_modules to .gitignore (as it can be added by accident while setting up linters).
- [ ] When you are sure you have made all the required changes you can submit for a re-review from your dashboard ✌🏿

medaminedev66 reviewed (approved):
Hi @tkz96 , 

## Status: Approved
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions:
- [ ] The Pull Request should have a descriptive title telling others what changes are inside. It would be better to change the title of your pull request to a descriptive title so your project is documented in a professional way.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tkz96 commented:
Thank you so much @prolajumokeoni and @medaminedev66
